### PR TITLE
Design/bootstrap

### PR DESCRIPTION
**Pull request by kasiaekiert**

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<footer class="page-footer font-small blue">
+  <div class="footer-copyright text-center py-3">© 2019 Copyright:
```
Comment (womanonrails) [inline app/views/layouts/_footer.html.erb]: There is no change which we did for this year. Did you push all changes to remote repository?

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNavDropdown">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/events">Events</a>
```
Comment (womanonrails) [inline app/views/layouts/_navbar.html.erb]: please use `link_to` method insted of `<a>`

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/places">Places</a>
```
Comment (womanonrails) [inline app/views/layouts/_navbar.html.erb]: The same here

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      </li>
+    </ul>
+  </div>
+
+  <div class="logininfo">
+      <% if user_signed_in? %>
+        <a href="#" class="btn btn-default btn-sm">
```
Comment (womanonrails) [inline app/views/layouts/_navbar.html.erb]: We removed this button classes this email label.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -4,22 +4,21 @@
     <title>Mapapp</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (womanonrails) [inline app/views/layouts/application.html.erb]: You add bootstrap from gem and directly in html. Please use only one version.

File: app/views/places/index.html.erb [hunk 6]
```diff
@@ -1,30 +1,28 @@
...
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Address</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class=container>
```
Comment (womanonrails) [inline app/views/places/index.html.erb]: missing `''`

File: app/views/places/index.html.erb [hunk 7]
```diff
@@ -1,30 +1,28 @@
-<h1>Places</h1>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 class="display-4">Hello, world!</h1>
+    <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+    <hr class="my-4">
+    <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
+    <a class="btn btn-primary btn-lg" href="/events" role="button">Show me EVENTS</a>
```
Comment (womanonrails) [inline app/views/places/index.html.erb]: use `link_to` method

File: app/views/places/index.html.erb [hunk 8]
```diff
@@ -1,30 +1,28 @@
...
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class=container>
+      <div class="row">
+        <% @places.each do |place| %>
+          <div class="col-md-4">
+            <h2><%= place.name %></h2>
+            <p><%= place.address %></p>
+            <p><a class="btn btn-secondary btn-xs" href="place" role="button">Show »</a></p>
```
Comment (womanonrails) [inline app/views/places/index.html.erb]: use `link_to` method

File: app/assets/javascripts/application.js [hunk 9]
```diff
@@ -13,4 +13,12 @@
...
 //= require turbolinks
+//= require jquery3
+//= require popper
+//= require bootstrap
+//= require bootstrap-sprockets
 //= require_tree .
+
+// $(document).ready(function() {
+//   $('.alert').alert();
+// });
```
Comment (womanonrails) [inline app/assets/javascripts/application.js]: Please remove code which is not used anymore.

File: app/assets/stylesheets/places.scss [hunk 10]
```diff
@@ -1,3 +1,34 @@
 // Place all the styles related to the places controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
```
Comment (womanonrails) [inline app/assets/stylesheets/places.scss]: Probably it will be better to move that to some common file. This changes are not only related with places but with all app. This comment is just about name of the file.

File: app/helpers/application_helper.rb [hunk 11]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def flash_class(level)
```
Comment (womanonrails) [inline app/helpers/application_helper.rb]: I would suggest to prepare method not only for class but for all flash message. Something like you have for `<a>` tag. In this case we use `link_to` method and this method generate whole HTML. The same you can do here with this flash message.

File: app/assets/javascripts/application.js [hunk 12]
```diff
@@ -16,6 +16,11 @@
 //= require jquery3
 //= require popper
 //= require bootstrap
+<<<<<<< HEAD
```
Comment (womanonrails) [inline app/assets/javascripts/application.js]: Right now you have conflicts to resolve.

File: app/assets/stylesheets/application.scss [hunk 13]
```diff
@@ -14,3 +14,7 @@
  *= require_self
  */
  @import "bootstrap";
+<<<<<<< HEAD:app/assets/stylesheets/application.scss
```
Comment (womanonrails) [inline app/assets/stylesheets/application.scss]: The same here. You have conflicts.

File: app/assets/stylesheets/places.scss (hunk 10 above)
Comment (womanonrails) [reply app/assets/stylesheets/places.scss]: This still need to be fixed
